aspnetcore/fundamentals/middleware/sample/Chain/project.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.1",
+      "type": "platform"
+    },
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
```

Comment:
VS 2017 only works with .csproj projects. The tooling in VS 2017 will not work for projec.json based projects.